webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -221,7 +221,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
               fetchExperiments={fetchExperiments}
               handleScroll={handleScroll}
               handleUpdateExperimentList={handleUpdateExperimentList}
-              height={height}
+              height={wholePageHeight - 140}
```

Comment:
What is 140?

Reply:
That's approximately the height of top section plus the height of bottom padding